_config.ts

```diff
@@ -161,44 +164,6 @@ site.copy("reference_gen/gen/node/page.css", "/api/node/page.css");
 site.copy("reference_gen/gen/node/styles.css", "/api/node/styles.css");
 site.copy("reference_gen/gen/node/script.js", "/api/node/script.js");
 
-site.process([".html"], (pages) => {
-  for (const page of pages) {
-    const document = page.document;
-    if (document) {
-      const tabGroups = document.querySelectorAll("deno-tabs");
```

Comment:
I'm all for removing this! Putting it in the client js works, but might incur an additional layout shift. That said if this makes working on the docs much more pleasent, I'm all for it.

Reply:
Yeah, there's a teeny tiny, split second shift, but I'm not sure if even notice it if I wasn't looking for it...